priority-queue-sf/PriorityQueue.Sender/Fabric.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Fabric;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ServiceFabric.Services.Runtime;
+using PriorityQueue.Shared;
+using System.Collections.ObjectModel;
+using System.Fabric.Description;
+using Microsoft.ServiceBus.Messaging;
+using System.Diagnostics;
+using System.Linq;
+
+namespace PriorityQueue.Sender
+{
+    /// <summary>
+    /// An instance of this class is created for each service instance by the Service Fabric runtime.
+    /// </summary>
+    internal sealed class Fabric : StatelessService
```

Comment:
Is this a common naming convention, to name the primary Stateless/StatefulService "Fabric"? I think I'd rather have the name suggest its function.